### DISCUSSION TITLE
Where are PCAPs Stored In Distributed Deployment

### DISCUSSION THREAD

**Discussion by Mfisher925** [Security-Onion-Solutions/securityonion#8134] [Unsupported Versions] [answered]
Hello,

Where are PCAPs stored in a distributed deployment?  Are they stored on the Foward node or Search nodes?

I need to expand the disk space available to allow for PCAPs to remain accessible for longer.

Thanks,

**Comment by mnasec** [comment, marked as answer]
In a distributed environment pcaps are stored on the forward node in the /nsm/pcap directory - see documentation: https://docs.securityonion.net/en/2.3/directory.html?highlight=pcap#nsm-pcap
